lib/providers/hlr-lookups-provider.js

```diff
@@ -44,7 +61,7 @@ HLRLookupsProvider.prototype.hlrLookup = function hlrLookup(number) {
     }
 
     if (!isProviderLookupEnabled) {
-      logger.info(logFunc('Returning dummy lookup result because actual calls are prohibited (env ENABLE_HLR_LOOKUPS not set to true)'));
+      logWarningIfRequired(logger, logFunc);
       if (!dummyResponse) {
         dummyResponse = require('./dummy_data/hlr-lookups-provider.json').data;
```

Comment:
Yeah it's a test thing. Because actual lookups cost money. You switch it on or off by setting the ENABLE_HLR_LOOKUPS env var.